dnsapi/dns_regru.sh

```diff
@@ -14,6 +14,8 @@ REGRU_API_URL="https://api.reg.ru/api/regru2"
 dns_regru_add() {
   fulldomain=$1
   txtvalue=$2
+  mydomain=${fulldomain#*_acme-challenge.}
```

Comment:
you should use the api to detect which part is the root zone.

https://github.com/acmesh-official/acme.sh/wiki/DNS-API-Dev-Guide#3-detect-which-part-is-your-root-zone

Reply:
Hello, Neil! But in line 33 

`%22subdomain%22:%22_acme-challenge%22`

That is therefore I thinking, that root = fulldomain - subdomain (in this case _acme-challenge).

Reply:
That was a mistake,   now is the time to fix it.

Reply:
any update ?

Reply:
> 
> 
> any update ?

Hello. I'm sorry, but I don't find how to got subdomain and rootzone from fulldomain. I look in https://www.reg.ru/support/help/api2 (russian) https://www.reg.com/support/help/api2 (english). There, requests are executed based on the domain name. 

I can try to ask a question in tech support of REGRU company how to do this.